mysql/test-users.sql

```diff
@@ -0,0 +1,31 @@
+USE orbitar_db;
+
+DELETE FROM user_invites WHERE child_id IN (SELECT user_id FROM users WHERE username LIKE 'testuser%');
+DELETE FROM invites WHERE code LIKE 'invite-testuser%';
+DELETE FROM users WHERE username LIKE 'testuser%';
+
+INSERT INTO users (username, password, email, gender, karma, name, registered_at)
+VALUES 
+  ('testuser1', '$2a$10$otySLU3y9XONpFnPqX.OfeNZrCeq4Ld.zC1RCXPCS4YAEGAlYEstO', 'test1@example.com', 0, 0, 'Test User 1', CURRENT_TIMESTAMP),
```

Comment:
I think there could be a small javascript generator in cypress that produces this sql, using configured passwords, and pipes it into mysql client, similar to this:

https://github.com/spaceshelter/orbitar/blob/9b5d39b375ba3810d2aed1b1275141f71e97679f/.github/workflows/restore-sanitized-db.yml#L92